app/javascript/src/components/Common/Menubar.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+import PropTypes from "prop-types";
+
+function Menubar({ title, options, setCategory }) {
+  const [isSearchCollapsed, setIsSearchCollapsed] = useState(true);
+
+  return (
+    <MenuBar showMenu title={title}>
+      {options?.heads?.map(head => (
+        <MenuBar.Block
+          label={head.title}
+          key={head.title}
+          count={head.count}
+          onClick={() => setCategory(head.title)}
+        />
+      ))}
+
```

Comment:
Please don't add unnecessary blank space in JSX! It's visually not helpful to add a blank line in JSX. Rather it's more confusing given that it's the presentation layer.

PS: It's a good practice to add blank in lines in JS to provide visual hierarchy. Here I am specifically talking about JSX.